internal/pkg/overlay/funcmap.go

```diff
@@ -130,3 +130,21 @@ func importSoftlink(lnk string) string {
 func softlink(target string) string {
 	return fmt.Sprintf("{{ /* softlink \"%s\" */ }}", target)
 }
+
```

Comment:
Following two points. 
* It just doesn't feel right that this is an internal function. It's run in templating so it's exported. Please name it UniqueField
* Document what the function does, like we document all functions. I like small riddles, but documentation is king.

Reply:
> It just doesn't feel right that this is an internal function.

I don't disagree; I just followed the example for all the existing functions. But I'm fine changing it.

Reply:
> Document what the function does, like we document all functions.

Done.